resources/home/dnanexus/utils/query.py

```
@@ -0,0 +1,487 @@
+            except Exception as exc:
+                # catch any errors that might get raised during querying
+                print(
+                    f"Error getting data for {concurrent_jobs[future]}: {exc}"
+                )
+
+    print(f"Found {len(calling_files.keys())} gCNV input / output files")
+
+    return calling_files
+
+
+def get_cnv_file_ids(reports, gcnv_dict):
```

Ruff (0.8.0)

`get_cnv_file_ids` is too complex (11 > 10)

(C901)